stage1/rootfs/usr_from_host/Makefile

```diff
@@ -1,13 +1,19 @@
 # do not include systemd in stage1, let's find it at run-time
 
-usr.done install: Makefile
+usr.done: Makefile install
+	cp -f install ../aggregate/install.d/00usr
+	touch usr.done
```

Comment:
I kept the same model as in [stage1/rootfs/usr_from_coreos/Makefile](https://github.com/coreos/rkt/blob/master/stage1/rootfs/usr_from_coreos/Makefile#L4) and [stage1/rootfs/usr_from_src/Makefile](https://github.com/coreos/rkt/blob/master/stage1/rootfs/usr_from_src/Makefile#L6).

It is because "../aggregate/install.d/00usr" can be written either by `usr_from_src`, `usr_from_coreos` or `usr_from_host`. We want to reference the `00usr` of the flavor we currently build, not just any `00usr` built by any flavor.